### PR TITLE
#3819 [Manager] Support the splittable layout

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3819 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Layout/AppLayout.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import SplitPane from 'react-split-pane';
+import AppBar from './AppBar';
+import * as context from 'context';
+import { Pipeline, Navigator } from 'components/Pipeline';
+import { DevToolDialog } from 'components/DevTool';
+import { Wrapper } from './AppLayoutStyles';
+
+const AppLayout = () => {
+  const { isOpen: isDevToolOpen } = context.useDevToolDialog();
```
Comment (saivirtue) [inline ohara-manager/client/src/components/Layout/AppLayout.js]: do we support the splittable layout for devToolDialog only?

Comment (eechih) [reply ohara-manager/client/src/components/Layout/AppLayout.js]: - AppBar 固定
- Navigator 可調整大小
- Pipeline 可調整大小
- DevTool 可調整大小，隱藏
- EventLog 可調整大小，隱藏

File: ohara-manager/client/src/components/Layout/AppLayoutStyles.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import styled, { css } from 'styled-components';
+
+export const Wrapper = styled.div(
+  ({ theme }) => css`
+    .Resizer {
+      box-sizing: border-box;
```
Comment (konekoya) [inline ohara-manager/client/src/components/Layout/AppLayoutStyles.js]: Didn't test this, but I think we could omit this as it's already inside our global CSS :)

Comment (eechih) [reply ohara-manager/client/src/components/Layout/AppLayoutStyles.js]: got it~

**Pull request merged**